lib/data-writer.js

```diff
@@ -121,16 +121,28 @@ class DataWriter {
     return uuid4().toUpperCase()
   }
 
+  static getElectricId() {
+    return '123456';
```

Comment:
The Electric user id should not be const value.

They will be the variable and pass to `output` method.

And this will be fixed on upcoming PR.